### PR TITLE
add some test and init

### PR DESCRIPTION
**Pull request by blaskodaniel**

### CODE REVIEW

File: src/app.tsx [hunk 1]
```diff
@@ -16,48 +13,16 @@ import HeaderPanel from './components/header'
...
-  // You won't need it on your project.
-  window: PropTypes.func,
-}
-
 /**
  * The main entry point of your app. You can start h@cking from here ;)
  */
-export const App: React.FunctionComponent = (props: Props) => {
+export const App: React.FunctionComponent = () => {
   // const usr = useCurrentUser()
```
Comment (zoltanbedi) [inline src/app.tsx]:
```suggestion
```

File: src/app.tsx [hunk 2]
```diff
@@ -16,48 +13,16 @@ import HeaderPanel from './components/header'
...
   // const repo = useRepository()
```
Comment (zoltanbedi) [inline src/app.tsx]:
```suggestion
```

File: src/components/list-panel.tsx [hunk 3]
```diff
@@ -57,7 +57,8 @@ const ListPanel: React.FunctionComponent<TodoItems> = ({ data, setData }) => {
   const toggleTask = async (task: Task) => {
     const currentIndex = data.indexOf(task)
     const newdata = [...data]
-    const newStatus = task.Status === Status.completed ? Status.active : Status.completed
+    const newStatus =
```
Comment (zoltanbedi) [inline src/components/list-panel.tsx]:
```suggestion
    const newStatus = task.Status?.[0] === Status.completed ? [Status.active] : [Status.completed]
```
With typescript 3.7 you can write it like this.

File: src/components/list-panel.tsx [hunk 4]
```diff
@@ -57,7 +57,8 @@ const ListPanel: React.FunctionComponent<TodoItems> = ({ data, setData }) => {
...
+      typeof task.Status !== 'undefined' && task.Status[0] === Status.completed ? [Status.active] : [Status.completed]
```
Comment (zoltanbedi) [inline src/components/list-panel.tsx]:
```suggestion
```

File: src/components/list-panel.tsx [hunk 5]
```diff
@@ -98,7 +99,7 @@ const ListPanel: React.FunctionComponent<TodoItems> = ({ data, setData }) => {
   const TodoList = data.map(d => {
     const labelId = `checkbox-list-label-${d.Id}`
     const deleteId = `checkbox-list-deletebtn-${d.Id}`
-    const classCompleted = d.Status === Status.completed ? 'comp' : ''
+    const classCompleted = typeof d.Status !== 'undefined' && d.Status[0] === Status.completed ? 'comp' : ''
```
Comment (zoltanbedi) [inline src/components/list-panel.tsx]: Same here. Please use conditional operator.

File: src/components/list-panel.tsx [hunk 6]
```diff
@@ -112,7 +113,7 @@ const ListPanel: React.FunctionComponent<TodoItems> = ({ data, setData }) => {
         <ListItemIcon>
           <Checkbox
             edge="start"
-            checked={d.Status == Status.completed}
+            checked={typeof d.Status !== 'undefined' && d.Status[0] == Status.completed}
```
Comment (zoltanbedi) [inline src/components/list-panel.tsx]: Same.

**Review by zoltanbedi — CHANGES_REQUESTED** [review]

File: src/components/list-panel.tsx (hunk 6 above)
Comment (blaskodaniel) [reply src/components/list-panel.tsx]: fixed

File: src/components/list-panel.tsx (hunk 5 above)
Comment (blaskodaniel) [reply src/components/list-panel.tsx]: fixed

**Review by zoltanbedi — APPROVED** [review]
🦸‍♂️

**Pull request merged**